composer.json

```diff
@@ -101,8 +98,8 @@
         "drupal/media_expire": "^2.4",
         "drupal/metatag": "^1.19",
         "drupal/metatag_async_widget": "^1.0-alpha2",
-        "drupal/paragraphs": "1.12",
-        "drupal/paragraphs_features": "^1.15",
+        "drupal/paragraphs": "^1.13",
```

Comment:
We should keep that at ^1.12 so people with different modules that might fail due to 1.13 will be able to stay on 1.12

Reply:
We have to use 1.13 because of PHP8.1 compatibility.